src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,103 @@
-import React from 'react';
+import React, { Dispatch, SetStateAction, useState } from 'react';
+import classNames from 'classnames';
+import { Comment, CommentData } from '../types/Comment';
+import { createComment } from '../api/comments';
+import { warningTimer } from '../utils/warningTimer';
+
+type Props = {
+  postId: number | undefined;
+  setComments: Dispatch<SetStateAction<Comment[]>>
+  setIsHasError: Dispatch<SetStateAction<boolean>>;
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  postId,
+  setComments,
+  setIsHasError,
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorName, setErrorName] = useState(false);
+  const [errorEmail, setErrorEmail] = useState(false);
+  const [errorBody, setErrorBody] = useState(false);
```

Comment:
Looks like one entity. Is it ok for you to make this error as one object state? Similar to newComment